packages/boilerplate/composables/src/getters/cartHelpers.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-vars, @typescript-eslint/no-unused-vars */
```

Comment:
+1 for getters

Reply:
Yes, I decided to add helpers at the end, because of the naming conflict with the getters from the composables, but you right, maybe I would change filenames to just `product, cart, ...etc` and in composables to `import {productStatics (productFn)} from './gettes'`. What do you think?

Reply:
With @filrak we decided to rename it to productPlainGetters

Reply:
I believe filename should be same as getters:
`productGetters -> geetters/productGetters.ts`
easy to find when you're interested with that particular getter. Just `product` wouldn't be enough and `productPlainGetters` wouldn't be any clue